tests/dragonfly/replication_test.py

```diff
@@ -1897,11 +1897,11 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     replica = df_local_factory.create(dbfilename=f"dump_{tmp_file_name}")
     df_local_factory.start_all([master, replica])
 
-    seeder = df_seeder_factory.create(port=replica.port, keys=30000)
+    seeder = df_seeder_factory.create(port=replica.port, keys=40000)
```

Comment:
also maybe if we you fixing this test, lets use static seeder here as this will make the test run much faster

Reply:
I will do this on this test here and when I get a chance I will make another one to clean up other cases :)